src/browser/BrowserPasswordGeneratorWidget.cpp

```diff
@@ -0,0 +1,346 @@
+/*
```

Comment:
Maybe. But the UI is different in the plugin than in the normal password generator. For example, there's no need for any generate buttons or the password text field etc. Of course I could give it a try.